Add option to append Segment IDs to tracking URLs in Mautic (#582)
mautic/user-documentation: docs/configuration/settings.rst

* **Identify visitors by IP** - Select **Yes** to use the IP address to identify Contacts. It's possible to track unidentified visitors with the same IP address as an existing Contact. This may result in undesirable outcomes with large Companies who use the same externally facing IP address.

* **Do Not Track 404 error for anonymous Contacts** - Select **Yes** to not track page hits on any 404 error page tracked by the tracking code. This option helps prevent filling your logs with hits from bots.

> * **Append Segment IDs to Tracking URLs** - Select **Yes** to have Mautic append Segment IDs to the tracking URLs in Emails sent from Mautic. This allows Mautic to track which Segment a Contact was part of when they clicked a link in an Email.

Reviewer: I think we better use present tense here?

```suggestion
* **Append Segment IDs to Tracking URLs** - Select **Yes** to enable Mautic to append Segment IDs to the tracking URLs in Emails sent from Mautic. This allows Mautic to track which Segment a Contact belongs to when they click on a link in an Email.
```

Reviewer: @escopecz any thought here?

Reviewer 2: Yep, better

* **Do Not Track 404 error for anonymous Contacts** - Select **Yes** to not track page hits on any 404 error page tracked by the tracking code. This option helps prevent filling your logs with hits from bots.

> * **Append Segment IDs to Tracking URLs** - Select **Yes** to enable Mautic to append Segment IDs to the tracking URLs in Emails sent from Mautic. This allows Mautic to track which Segment a Contact belongs to when they click on a link in an Email.

Reviewer: ```suggestion
* **Append Segment IDs to Tracking URLs** - Select **Yes** to enable Mautic to append Segment IDs to the tracking URLs in Emails sent from Mautic. This allows Mautic to track which Segment a Contact belongs to when they click a link in an Email.
```
